serverlessworkflow/modules/ROOT/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,154 @@
+= Build and deploy in Kubernetes a {context} application using the {product_name} {context} Operator
+
+:compat-mode!:
+// Metadata:
+:description: Build and deploy using the Kogito Serverless Workflow Operator a serverless workflow application
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube
+// links
+:minikube_url: https://minikube.sigs.k8s.io
+:kogitoserverlessoperator_url: https://github.com/kiegroup/kogito-serverless-operator/
+:kogitogreetinexample_url: https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-examples/serverless-workflow-greeting-quarkus
+:kanikoissue_url: https://github.com/GoogleContainerTools/kaniko/issues/2201
+
+This document describes how to build and deploy your workflow application using a local Kubernetes cluster, such as link:{minikube_url}[Minikube], along with the link:{kogitoserverlesosperator_url}[{product_name} Serverless Operator].
+
+Using the {product_name} Serverless Operator] you will be able both to build and deploy a {product_name} {context} application so that you only need the workflow definition.
```

Comment:
```suggestion
Using the {product_name} Serverless Operator] you will be able to build and deploy a {product_name} {context} application so that you only need the workflow definition.
```